src/libs/actions/Task.js

```diff
@@ -259,10 +265,39 @@ function completeTask(taskReportID, taskTitle) {
         },
     ];
 
-    // Update optimistic data for parent report action
-    const optimisticParentReportData = ReportUtils.getOptimisticDataForParentReportAction(taskReportID, completedTaskReportAction.created, CONST.RED_BRICK_ROAD_PENDING_ACTION.ADD);
-    if (!_.isEmpty(optimisticParentReportData)) {
-        optimisticData.push(optimisticParentReportData);
+    // Update optimistic data and failure data for parent report action
+    if (taskReport.parentReportActionID) {
+        let parentReportAction = ReportActionsUtils.getParentReportAction(taskReport);
+        if (parentReportAction && parentReportAction.reportActionID) {
```

Comment:
Is this condition required?

Reply:
I think so. If we are not able to find assignee parent report action for some reasons, parentReportAction is {}. We need to check it